src/objects/jira_base.py

```diff
@@ -148,6 +147,23 @@ def create_issue(
 
         return issue
 
+    def add_attachment_to_issue(self, issue: Issue, attachment_path: str) -> None:
+        """
+        Add and upload the attachment from attachment_path to the Jira Issue object provided.
+
+        Args:
+            issue (Issue): The Jira issue the attachment should be added to.
+            attachment_path (str): The path of the file to upload as an attachment to the issue.
+
+        Returns:
+            None
+        """
+        try:
+            self.connection.add_attachment(issue=issue.key, attachment=attachment_path)
+            self.logger.info(f"Attachment {attachment_path} has been uploaded to {issue}")
+        except JIRAError as e:
```

Comment:
Can you please do not use one later words? `e`

Reply:
> Can you please do not use one later words? `e`

The use of one-letter variables is a very common coding style choice in Python, especially in the case of context managers. They are also used frequently in the [official python documentation](https://docs.python.org/3/tutorial/errors.html) examples for exception handling (see sections [¶8.9](https://docs.python.org/3/tutorial/errors.html#raising-and-handling-multiple-unrelated-exceptions), [¶8.10](https://docs.python.org/3/tutorial/errors.html#enriching-exceptions-with-notes)) and the [PEP 8 style guide](https://peps.python.org/pep-0008/). They are commonly used to indicate that a variable is intended to be short-lived and tightly-scoped.

For this suggested change and others related to coding style, can we instead open an issue to document the preferred coding style and add it to the contribution guide, especially those that break away from common convention?